admin-data-explorer-play-server/src/main/scala/net/wiringbits/webapp/utils/admin/tasks/DataExplorerConfigValidatorTask.scala

```diff
@@ -57,6 +58,13 @@ class DataExplorerConfigValidatorTask @Inject() (
   private def validatePrimaryKeyFieldName(idFieldName: String, fields: List[TableField]): Unit = {
     val exists = fields.exists(_.name == idFieldName)
     if (exists) ()
-    else throw new RuntimeException(s"The provided id on DataExplorer settings doesn't exists: ${idFieldName}")
+    else throw new RuntimeException(s"The provided id on DataExplorer settings doesn't exists: $idFieldName")
+  }
+
+  private def validateHiddenColumns(columns: List[String], tableFields: List[TableField]): Unit = {
```

Comment:
I'd call this `validateVisibleFields` instead, then, as arguments use `targetFields` and `hiddenFields`

Also, add tests to verify this.